src/services/storage/drivers/redis/types.ts

```diff
@@ -0,0 +1,6 @@
+import { config } from '../../../../config/index.js'
```

Comment:
This file is strictly for types. Don't import the config here.

If you need the urlLifeTime from the config, just copy it in the config module.